python/ray/serve/scripts.py

```diff
@@ -100,6 +107,35 @@ def process_dict_for_yaml_dump(data):
     return data
 
 
+def print_status_for_app(name: str, application: ApplicationDetails):
+    status = json.loads(
```

Comment:
Instead of calling `json.loads()` and `.json()`, could you call `ServeStatusSchema(...).dict()`?

Reply:
Sounds good, thanks!